LiteDB/Engine/Pages/BasePage.cs

```diff
@@ -225,7 +225,6 @@ public void MarkAsEmtpy()
             this.IsDirty = true;
 
             this.PageType = PageType.Empty;
-            this.ItemsCount = 0;
```

Comment:
Why this line was removed?

Reply:
This is old changes. You have the same line repeated twice https://github.com/blowin/LiteDB/blob/6c9d53a615216d615ada61ede7b7b9ab8f90ab4e/LiteDB/Engine/Pages/BasePage.cs#L151